docs/developer/database-access.md

```diff
@@ -12,19 +12,32 @@ to get a `psql` shell on the sandbox environment's database.
 
 ## Running Migrations
 
+### On Local
+
```

Comment:
maybe add context here, such as why you may want to do it. (ie you changed a model, or have pulled in new code and are seeing the warning when you do docker-compose up).